src/libs/Analyzer.js

```diff
@@ -0,0 +1,58 @@
+const { DataSetBuilder } = require('../DataPipeline/DataSetBuilder');
+
+class Analyzer {
+  constructor(owner, repo) {
+    this.owner = owner;
+    this.repo = repo;
+  }
+
+  async getDataPoints() {
+    const prArray = [];
+
+    const dataSetBuilder = new DataSetBuilder(this.owner, this.repo);
+    const data = await dataSetBuilder.getFinalData();
+    data.forEach((prDetails, prKey, map) => {
+      const {
+        numberOfBugs,
+        numberOfCommits,
+        merged_at,
+        closed_at,
+        created_at,
+      } = prDetails;
+      if (!(numberOfBugs && numberOfCommits && merged_at && closed_at && created_at)) return;
+      prArray.push(prDetails);
+    });
+
+    if (prArray.length === 0) return [];
+
+    // Sort by created_at in ascending order
+    const sortedPRs = prArray.sort((a, b) => new Date(a.created_at) - new Date(b.created_at));
+    const dateRepoCreated = sortedPRs[0].created_at;
```

Comment:
so we're going with the date of when repo was created as the date of the first PR's creation? is that accurate?

Reply:
I think that's gonna be as accurate as it's gonna get since Patrick wasn't able to get the Repo creation date